### PR TITLE
Add Pipeline Settings

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Review by LinuxSuRen — APPROVED** [review]

File: content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md [hunk 1]
```diff
@@ -1,9 +1,203 @@
...
+- You need to create a workspace, a DevOps project and an account (`project-regular`). This account must be invited to the DevOps project with the `operator` role. For more information, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
+- You need to [enable the KubeSphere DevOps System](../../../pluggable-components/devops/).
+
+## Pipeline Settings
+
+### Basic Information
+
+![basic-info-tab](/images/docs/devops-user-guide/using-devops/pipeline-settings/basic-info-tab.png)
+
+On the **Basic Info** tab, you can customize the following information:
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md:22]: Basic Info
->
Basic Information

The latest console UI is Basic Information

File: content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md [hunk 2]
```diff
@@ -1,9 +1,203 @@
...
+
+  {{</ tab >}}
+
+  {{</ tabs >}}
+
+### Advanced Settings with A Code Repository Selected
+
+If you selected a code repository, you can customize the following configurations on the **Advanced Settings** tab:
+
+**Branches Settings**
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md:80]: Branches Settings
->
Branch Settings

Change the UI text in the console repository.

File: content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md [hunk 3]
```diff
@@ -1,9 +1,203 @@
...
+
+![script-path](/images/docs/devops-user-guide/using-devops/pipeline-settings/script-path.png)
+
+The field of **Script Path** specifies the Jenkinsfile path in the code repository. It indicates the repository’s root directory. If the file location changes, the script path also needs to be changed.
+
+**Scan Repo Trigger**
+
+![scan-repo-trigger](/images/docs/devops-user-guide/using-devops/pipeline-settings/scan-repo-trigger.png)
+
+You can check **Enable regular expressions, ignoring names that do not match the provided regular expression (including branches and PR, etc.)** to specify a regular expression as the trigger for scanning the code repository.
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md:139]: (including branches and PR, etc.)
->
(including branches and PRs)

Change the UI text in the console repository.

File: content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md [hunk 4]
```diff
@@ -1,9 +1,203 @@
...
+
+### Advanced Settings with No Code Repository Selected
+
+If you don't select a code repository, you can customize the following configurations on the **Advanced Settings** tab:
+
+**Build Settings**
+
+![build-settings](/images/docs/devops-user-guide/using-devops/pipeline-settings/build-settings.png)
+
+**Discard old builds** determines when the build records under the branch will be discarded. The build record includes the console output, archive artifacts, and other metadata related to a particular build. Keeping less builds saves disk space used by Jenkins. KubeSphere provides two options to determine when old builds are discarded:
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md:171]: archive artifacts
->
archived artifacts

Keeping less builds saves disk space used by Jenkins.
->
Keeping fewer builds saves disk space used by Jenkins.

File: content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md [hunk 5]
```diff
@@ -1,9 +1,203 @@
...
-keywords: 'KubeSphere, Kubernetes, Docker, Jenkins, pipelines'
+keywords: 'KubeSphere, Kubernetes, Docker, Jenkins, Pipelines'
 description: 'Understand various pipeline properties in a DevOps project.'
 linkTitle: "Pipeline Settings"
 weight: 11280
 ---
 
-TBD.
\ No newline at end of file
+When you create a pipeline, you can customize its configurations through various settings. This document illustrates the settings when creating a pipeline.
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md:9]: This document illustrates the settings when creating a pipeline.
->
This document illustrates these settings in detail.

File: content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md [hunk 6]
```diff
@@ -1,9 +1,203 @@
...
+
+  {{</ notice >}}
+
+- **No concurrent builds**. If you check this option, you cannot run multiple builds concurrently.
+
+**Parametric Build**
+
+![parametric-build](/images/docs/devops-user-guide/using-devops/pipeline-settings/parametric-build.png)
+
+The parameterized build process allows you to pass in one or more parameters when you start to run a pipeline. KubeSphere provides five types of parameters by default, including **String**, **Text**, **Boolean**, **Choice**, and **Password**. When parameterizing a project, the build is replaced with a parameterized build, which prompts the user to enter a value for each defined parameter.
```
Comment (Sherlock113) [inline content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md:189]: When parameterizing a project
->
When you parameterize a project

File: content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md (hunk 2 above)
Comment (Felixnoo) [reply content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md:80]: UI changed. Please take a look at [the PR](https://github.com/kubesphere/console/pull/1762/commits/114049c675aaa768c807d526e7a8793e10e9a0e1).

**Pull request merged**